python_scripts/03_basic_preprocessing_categorical_variables.py

```diff
@@ -16,6 +16,12 @@
 
 # %%
 # TODO add intro with objectives
+## Working with both numerical & categorical variables
+
+# In this notebook, we will present:
+# * typical ways to deal with categorical variables
+# * how to train a predictive model on different types of data
+# (i.e. numerical and categorical)
 
 # ## [markdown]
```

Comment:
This should be `# %% [markdown]` here right?

Not done by you but while I am at it I thought I would report it ...